messagepackhubprotocol.go

```diff
@@ -43,10 +43,13 @@ func (m *messagePackHubProtocol) parseMessage(buf *bytes.Buffer) (interface{}, e
 	if err != nil {
 		return nil, err
 	}
-	// Ignore Header
-	_, err = decoder.DecodeMap()
-	if err != nil {
-		return nil, err
+	// Ignore Header for all messages, except ping and close message that have no header
+	// see message spec at https://github.com/aspnet/SignalR/blob/master/specs/HubProtocol.md
+	if msgType8 != 6 && msgType8 != 7 {
```

Comment:
https://github.com/dotnet/aspnetcore/blob/main/src/SignalR/docs/specs/HubProtocol.md#message-headers (which should be the reference as aspen is outdated) says only the ping message has no header.
But https://github.com/aspnet/SignalR/blob/master/specs/HubProtocol.md#message-headers states the same.